noSelectStar/noSelectStar.go

```diff
@@ -0,0 +1,57 @@
+package noSelectStar
+
+import (
+	"go/ast"
+	"go/token"
+	"strings"
+
+	"golang.org/x/tools/go/analysis"
+)
+
+var Analyzer = &analysis.Analyzer{
+	Name: "noSelectStar",
+	Doc:  "checks for SQL queries containing SELECT * which breaks forwards compatibility",
+	Run:  run,
+}
+
+func run(pass *analysis.Pass) (interface{}, error) {
+	inspect := func(node ast.Node) bool {
```

Comment:
Can we declare these as external functions? It looks more readable that way. Also, would `inspectStrings` be a better name here?

Reply:
I've simplified this to just a single `inspect` -- this matches the pattern elsewhere in this repo, and also avoids having to keep `pass` in scope for the callback.